api/src/uvisor-input.S

```diff
@@ -84,6 +85,11 @@ uvisor_config:
     /* pointer to __uvisor_box_context */
     .long __uvisor_box_context
 
+    /* start and end address and page size of page heap */
```

Comment:
Can you replace this comment with something like `/* Page allocator heap boundaries and size */`? See [this commit](https://github.com/ARMmbed/uvisor/pull/233/commits/d1128a81846fed3f584b8677bbd46393a8ae651d) for reference.